src/core/types/index.js

```diff
@@ -19,3 +19,12 @@ export {isArray} from './array';
 export {isEnumValue} from './enum';
 export {isString} from './string';
 export {isObject} from './object';
+
+/**
+ * Determines if value is an ELement
+ * @param {*} value
+ * @return {boolean}
+ */
+export function isElement(value) {
+  return value?.tagName == /* Node.ELEMENT_NODE */ 1;
```

Comment:
```suggestion
  return value?.nodeType == /* Node.ELEMENT_NODE */ 1;
```

Reply:
Fixed in separate commit. How did you even end up seeing this PR? I was just testing its bundle-size impact before considering sending it out